### PR TITLE
Export users to Drupal.

### PR DESCRIPTION
**Pull request by sergii-tkachenko**
This PR provides Artisan command that migrates CGG users to Drupal using Services API.
Usage example:
```
php artisan migrate:drupal http://dev.dosomething.org:8888/api/v1/ US
```

### CODE REVIEW

**Comment by sergii-tkachenko** [comment]
@angaither NOTE: I removed `$last_user_imported` functionality.

**Comment by sergii-tkachenko** [comment]
Just checked, maximum users to migrate is on Canada, `3546`. Should I split it in batches or should be OK?

File: app/commands/DrupalMigrate.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+      $fields =  [
+        'email'                    => $user->email,
+        'birthdate'                => $user->birthdate,
+        'first_name'               => $user->first_name,
+        'user_registration_source' => 'CGG',
+      ];
+
+      // Optional.
+      if (!empty($user->phone)) {
```
Comment (angaither) [inline app/commands/DrupalMigrate.php]: we don't have phone numbers for any international users, so this isn't needed

Comment (sergii-tkachenko) [inline app/commands/DrupalMigrate.php]: Hm. As I know, it's required on UK.

Comment (angaither) [inline app/commands/DrupalMigrate.php]: But we didn't collect any phone numbers for international users in this app

**Comment by angaither** [comment]
@sergii-tkachenko I would leave in the `last-imported` so that if this fails in the middle it doesn't try to re-migrate the users that were already migrated

**Comment by sergii-tkachenko** [comment]
Ok, will do.

**Comment by angaither** [comment]
@sergii-tkachenko you shouldn't need to split this in batches, it ran ok when I migrated allll of the users into northstar, but that's what the `last-imported` thing was for, way easier than batches ;)

**Comment by sergii-tkachenko** [comment]
Depends on DoSomething/dosomething#4418. It should be deployed before this can be executed on production.

**Comment by sergii-tkachenko** [comment]
@angaither Please re-review.

**Comment by angaither** [comment]
@sergii-tkachenko looks good

I think the last import stuff [could be simplified](https://github.com/DoSomething/voting-app/blob/master/app/Console/Commands/Northstar.php), but if it works, that's fine

**Comment by sergii-tkachenko** [comment]
The difference is that your last import requires migration, my doesn't. And mine actually becomes a select condition so less user models will be loaded.

**Pull request merged**